compile error when opencensus-contrib-spring is used with maven

When opencensus-spring-contrib is used with in maven builds, it fails to build due to following error. 

[ERROR] Failed to execute goal on project oc-spring-servlet-example: Could not resolve dependencies for project io.opencensus:oc-spring-servlet-example:jar:0.23.0-SNAPSHOT: Failure to find org.springframework.cloud:spring-cloud-build:jar:1.3.4.RELEASE in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

The error is because of spring-cloud-build package is of type 'pom'.

### What version of OpenCensus are you using?
0.23.0-SNAPSHOT

### What JVM are you using (`java -version`)?
java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed mode)

--- a/build.gradle
+++ b/build.gradle
@@ -214,7 +214,7 @@ subprojects {
                 spring_aspects: "org.springframework:spring-aspects:${springVersion}",
                 spring_boot_starter_web: "org.springframework.boot:spring-boot-starter-web:${springBootVersion}",
                 spring_boot_starter_web2: "org.springframework.boot:spring-boot-starter-web:${springBoot2Version}",
-                spring_cloud_build: "org.springframework.cloud:spring-cloud-build:${springCloudVersion}",
+                spring_cloud_build: "org.springframework.cloud:spring-cloud-build:${springCloudVersion}@pom",
                 spring_cloud_starter_sleuth: "org.springframework.cloud:spring-cloud-starter-sleuth:${springCloudVersion}",
                 spring_context: "org.springframework:spring-context:${springVersion}",
                 spring_context_support: "org.springframework:spring-context-support:${springVersion}",

